### PR TITLE
add pause shortcut shift + sapce (#5301)

### PR DESCRIPTION
**Pull request by tjrdnjs1609**
add shortcut shift + space for pause on SongEditor.cpp
it only work on playing state.
when it paused, shift + space does not work.

### CODE REVIEW

**Comment by tjrdnjs1609** [comment]
i solve https://github.com/LMMS/lmms/issues/5301 this problem.
anyone review my code??? is less then 10 line...

**Comment by PhysSong** [comment]
Sorry for the late reply. You'd better look into `Editor`, the base class of `SongEditor`. That will make the shortcut available for other editors such as the beat/bassline editor.
Also, `Song::togglePause()` looks better for pausing the playback.

**Comment by tjrdnjs1609** [comment]
@PhysSong thanks for reply. i'll check again.

**Comment by PhysSong** [comment]
@tjrdnjs1609 Are you still interested in this? If not, I'm going to close this so anyone else can code fixes for #5301. Otherwise, I can help you finish this stuff.

File: src/gui/editors/SongEditor.cpp [hunk 1]
```diff
@@ -523,6 +523,15 @@ void SongEditor::keyPressEvent( QKeyEvent * ke )
...
 	}
+	else if ( ke->key() == Qt::Key_Space && ke->modifiers() & Qt::ShiftModifier )
+	{
+		tick_t t = m_song->currentTick();
+		if ( t >= 0 && m_song->isPlaying() )
+		{
+			m_song->stop();
+			m_song->setPlayPos( t, Song::Mode_PlaySong );
+		}
+	}
```
Comment (russiankumar) [inline src/gui/editors/SongEditor.cpp:534]: I think this code is okay. I don't have write access either ways, but I think you should rebase to fix all those other files that are messed up in the PR, and I think you'll be ready for a merge (which I can't do).

**Comment by PhysSong** [comment]
Closed in favor of #5554.

**Pull request closed without merging**